lib/MusicBrainz/Server/Data/Utils.pm

```diff
@@ -341,6 +342,14 @@ sub trim {
     return $t;
 }
 
+sub trim_comment {
+    my $t = shift;
+
+    $t =~ s/^\s*\((.+)\)\s*$/$1/;
```

Comment:
It should probably trim spaces within parentheses, and unpaired parentheses too:
```suggestion
    $t =~ s/^[\s\(]*(.+)[\)\s]*$/$1/;
```

Reply:
If what you mean with unpaired parentheses is stuff like "whatever)", then it shouldn't because that'd also break a valid disambiguation like "some stuff (more stuff)".

Reply:
Once the parentheses are removed, then this does trim($t), so any newly-trimmable whitespace will still get trimmed.